examples_gbs/run_tutorial_max_clique.py

```diff
@@ -0,0 +1,155 @@
+are of interest in determining how the molecule interacts with the protein.
+
+The first step is to import the Strawberry Fields GBS module and external dependencies:
+"""
+from strawberryfields.gbs import data, plot, sample, clique
+import numpy as np
+import networkx as nx
+import plotly
+
+##############################################################################
+# The adjacency matrix of the TACE-AS graph can be loaded from the ``data`` module and the
+# graph can be visualized using the :mod:`~gbs.plot` module:
+
+TA = data.TaceAs()
+A = TA.adj
+TA_graph = nx.Graph(A)
+graph_fig = plot.plot_graph(TA_graph)
+plotly.offline.plot(graph_fig, filename="TACE-AS.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/TACE-AS.html
+
+##############################################################################
+# Can you spot any cliques in the graph? It's not so easy using only your eyes! The TACE-AS graph
+# is sufficiently small that all cliques can be found by performing an exhaustive search over
+# all subgraphs. For example, below we highlight a small *maximal* clique, i.e., a clique
+# not contained inside another clique:
+
+maximal_clique = [4, 11, 12, 18]
+maximal_fig = plot.plot_graph(TA_graph, maximal_clique)
+plotly.offline.plot(maximal_fig, filename="maximal_clique.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/maximal_clique.html
+
+##############################################################################
+# We'll now use the :mod:`~clique` module to find larger cliques in the graph. We can make
+# use of the pre-generated samples from the TACE-AS graph in the ``data`` module and post-select samples with a specific number of clicks. For
+# this tutorial, we'll look at samples with eight clicks, of which there are a total of 1,984:
+
+TA_subgraphs = sample.to_subgraphs(TA, TA_graph)
+samples = sample.postselect(TA_subgraphs, 8, 8)
+print(len(samples))
+
+##############################################################################
+# GBS produces samples that correspond to subgraphs of high density. For fun, let's confirm this
+# by comparing the average subgraph density in the GBS samples to uniformly generated samples:
+
+GBS_dens = []
+u_dens = []
+
+for s in samples:
+    uniform = list(np.random.choice(24, 8, replace=False))  # generates uniform sample
+    GBS_dens.append(nx.density(TA_graph.subgraph(s)))
+    u_dens.append(nx.density(TA_graph.subgraph(uniform)))
+
+print('GBS mean density = {:.4f}'.format(np.mean(GBS_dens)))
+print('Uniform mean density = {:.4f}'.format(np.mean(u_dens)))
+
+##############################################################################
+# Those look like great GBS samples 💪! To obtain cliques, we shrink the samples by greedily

```

Comment:
I fully endorse emoji's in tutorials :muscle:

Reply:
philosophical question: is it _GBS samples_ or _GB samples_? 😄

Reply:
"Gaussian boson sampling samples"

Reply:
Oddly enough it reads okay in my head.